tests/test_extension_image.py

```diff
@@ -51,5 +51,6 @@ def test_multiple_snapshot_extensions(snapshot):
     """
     assert actual_svg == snapshot(extension_class=SVGImageSnapshotExtension)
     assert actual_svg == snapshot  # uses initial extension class
+    assert snapshot._extension is not None
```

Comment:
This is testing internals. Is there a user facing test?

Reply:
There's no user effect only that slight performance drawback of generating the extension instance on each assertion

Reply:
Bit concerned about coupling the internal implementation with the test cases, but am okay with this as a temporary measure until we have perf benchmarking